variables.tf

```diff
@@ -1,5 +1,22 @@
-variable "name" {
-  type = string
-  default = "test"
+variable "prefix" {  
 }
 
+variable "resource_group_name" {
```

Comment:
Can you add a description on these variables as well please

Reply:
description added